image/gamma_correction/main.py

```diff
@@ -0,0 +1,14 @@
+from gamma_correction import *
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.